pkg/runcfanotify/runcfanotify.go

```diff
@@ -119,6 +119,7 @@ var runcPaths = []string{
 	"/bin/runc",
 	"/usr/bin/runc",
 	"/usr/sbin/runc",
+	"/usr/local/bin/runc",
```

Comment:
Can you please sync the changes with [container-hook](https://github.com/inspektor-gadget/inspektor-gadget/blob/main/pkg/container-hook/tracer.go#L141) as well? We will have to do it till https://github.com/inspektor-gadget/inspektor-gadget/issues/1831 is resolved.

Reply:
Good catch!